src/Faker/Generator.php

```diff
@@ -14,6 +14,7 @@
  * @method string title(string $gender = null)
  * @property string $titleMale
  * @property string $titleFemale
+ * @property string $ssn
```

Comment:
only generally available formatters (defined in `src/Faker/Provider/Person.php`) should appear here. Otherwise, people using other locales, with no `ssn` formatter, will have a suggestion for a non-existing method.